packages/shared/src/hooks/useSquadChecklist.tsx

```diff
@@ -151,60 +152,56 @@ const useSquadChecklist = ({
           component: (props) => <GoPublicStep {...props} squad={squad} />,
         },
         actions: getPublicActions(squad),
+        condition: () => {
+          const isGoPublicDismissed =
+            isChecklistReady &&
+            checkHasCompleted(ActionType.HidePublicSquadStep);
+          return !squad.public && !isGoPublicDismissed;
+        },
```

Comment:
We should completely remove this item: https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1083408386/RFC-058+-+Public+Squads+GA#Public-Squad-eligibility